src/redis/cache_impl.go

```diff
@@ -1,240 +1,15 @@
 package redis
 
 import (
-	"math"
 	"math/rand"
 
 	"github.com/coocood/freecache"
-	pb "github.com/envoyproxy/go-control-plane/envoy/service/ratelimit/v3"
-	"github.com/envoyproxy/ratelimit/src/assert"
-	"github.com/envoyproxy/ratelimit/src/config"
 	"github.com/envoyproxy/ratelimit/src/limiter"
 	"github.com/envoyproxy/ratelimit/src/server"
 	"github.com/envoyproxy/ratelimit/src/settings"
-	"github.com/envoyproxy/ratelimit/src/utils"
-	"github.com/golang/protobuf/ptypes/duration"
 	logger "github.com/sirupsen/logrus"
-	"golang.org/x/net/context"
 )
 
-type rateLimitCacheImpl struct {
```

Comment:
Can you please split this code movement part of the PR out into another PR so this one is easier to review? Thank you.

/wait

Reply:
Hi @mattklein123
Created PR for code movement part
https://github.com/envoyproxy/ratelimit/pull/194